apps/docs-snippets/src/guide/cookbook/transferring-assets.test.ts

```diff
@@ -26,7 +26,7 @@ describe(__filename, () => {
 
   it('should successfully transfer asset to another account', async () => {
     // #region transferring-assets-1
-    // #context import { Wallet, BN, BaseAssetId } from 'fuels';
+    // #addImport: Wallet, BN, BaseAssetId
```

Comment:
Is there any particular reason why you went with `#addImport a, b, c`?

I think a more standardized syntax would be more intuitive:

```ts
// #import { Wallet, BN, BaseAssetId };
```

Reply:
@arboleya No particular reason.

But yeah, the suggested approach seems more intuitive to me as well.